### DISCUSSION TITLE
When using Carplay there needs to be a home button in the search window

### DISCUSSION THREAD

**Discussion by Raycock2** [organicmaps/organicmaps#2474] [Ideas]
When using carplay there needs to be a home button in the search window to be able get to home without typing in the whole address.

**Comment by biodranik** [comment]
Thanks, I've created #2476, you can share your ideas about the look/design/positioning there.

**Reply by Raycock2** [reply]
![Home1](https://user-images.githubusercontent.com/100967885/166304452-6bea6241-40a1-48e0-b56f-3b958e33ba16.jpg)
![Home2](https://user-images.githubusercontent.com/100967885/166304514-b3cc9b78-1fec-4503-891e-872eb02ab01a.jpg)
![home3](https://user-images.githubusercontent.com/100967885/166304565-8908bc8b-adb6-4d26-bb1d-32839e5384cb.jpg)

**Reply by biodranik** [reply]
It was discussed in #1133
